robot/CMakeLists.txt

```diff
@@ -145,19 +145,12 @@ if(TARGET CAF::net)
         --variable SSL_PATH:${CMAKE_CURRENT_SOURCE_DIR}
         "${CMAKE_CURRENT_SOURCE_DIR}/web_socket/echo.robot")
 
-  caf_add_test_executable(
-    caf-robot-http-client-driver
-    SOURCES
-      http/client-driver.cpp
-    DEPENDENCIES
-      CAF::net)
-
   add_test(
     NAME "robot-http-client"
     COMMAND
       ${Python_EXECUTABLE}
         -m robot
-        --variable BINARY_PATH:$<TARGET_FILE:caf-robot-http-client-driver>
+        --variable BINARY_PATH:$<TARGET_FILE:client>
```

Comment:
That target only exists when building the examples in the first place. I think the `if` statement above should read as `if(TARGET CAF::net AND CAF_ENABLE_EXAMPLES)`.

Not a request for this PR, more of a general observation. I think we should rename the example targets. A binary called `client` is too ambiguous.

Reply:
Agree on this, we should consider prefixing the targets by path.